psi4/src/psi4/libmints/basisset.cc

```diff
@@ -192,24 +191,19 @@ void BasisSet::print(std::string out) const
 {
     std::shared_ptr <psi::PsiOutStream> printer = (out == "outfile" ? outfile :
                                                    std::shared_ptr<OutFile>(new OutFile(out)));
-    if(basistype_ == GaussianBasis) {
-        printer->Printf("  Basis Set: %s\n", name_.c_str());
-        printer->Printf("    Blend: %s\n", target_.c_str());
-        printer->Printf("    Number of shells: %d\n", nshell());
-        printer->Printf("    Number of basis function: %d\n", nbf());
-        printer->Printf("    Number of Cartesian functions: %d\n", nao());
-        printer->Printf("    Spherical Harmonics?: %s\n", has_puream() ? "true" : "false");
-        printer->Printf("    Max angular momentum: %d\n\n", max_am());
-    } else if(basistype_ == ECPBasis) {
-
+    printer->Printf("  Basis Set: %s\n", name_.c_str());
+    printer->Printf("    Blend: %s\n", target_.c_str());
+    printer->Printf("    Number of shells: %d\n", nshell());
+    printer->Printf("    Number of basis function: %d\n", nbf());
+    printer->Printf("    Number of Cartesian functions: %d\n", nao());
+    printer->Printf("    Spherical Harmonics?: %s\n", has_puream() ? "true" : "false");
+    printer->Printf("    Max angular momentum: %d\n\n", max_am());
+    if(has_ECP()) {
         printer->Printf("  Core potential: %s\n", name_.c_str());
-        printer->Printf("    Blend: %s\n", target_.c_str());
-        printer->Printf("    Number of shells: %d\n", nshell());
-        printer->Printf("    Number of terms: %d\n", nprimitive());
+        printer->Printf("    Number of shells: %d\n", n_ecp_shell());
+        printer->Printf("    Number of terms: %d\n", n_ecp_primitive());
```

Comment:
I've resorted to calling pieces of a gbs file "blocks", if that appeals. Only other info I can find is Cfour output:

```
 ECP: maximum angular momentum =                     2
 ECP: maximum angular projector=                     2
 total number of basis functions                    39
```
Should the number of electrons covered by ECP be printed? (EDIT: guess I just need to scroll)